### PR TITLE
Edit survey with sections 

### PR DESCRIPTION
**Pull request by martinagvilas**
Hi, I edited the survey and added sections (and some questions) inspired by the links provided in #91.

I also changed the employment questions that refered to IT jobs to refer to more general aspects, to avoid assuming background.

Let me know what you think.

### CODE REVIEW

File: SURVEY.md [hunk 1]
```diff
@@ -147,12 +79,94 @@ Add all the questions that pops in your mind we will rank them later on.**
...
+- Every other day
+- Once a week
+- Once a month
+- Never
+
+### 3.2. Do you use pandas at work?
+- yes
+- no
+
+### 3.3. What do you use Pandas for? (multiple choice/single choice)
```
Comment (datapythonista) [inline SURVEY.md]:
```suggestion
### 3.3. What do you use pandas for? (multiple choice/single choice)
```

File: SURVEY.md [hunk 2]
```diff
@@ -147,12 +79,94 @@ Add all the questions that pops in your mind we will rank them later on.**
...
 
+### 4.3. Is this the first open-source project you contribute to?
+- yes
+- no
 
+## 5. Scholarship information
 
+### 5.1. Would you be interested in receiving a scholarship for your contribution to the pandas-mentoring team 2019?
+- yes
+- no
```
Comment (datapythonista) [inline SURVEY.md:175]: For me (and the pandas core devs who can find grant opportunities) would be useful to also know information like how many hours per week could be spent in the internship. And it's trickier, but information on how much money we need for the scholarships. Programs like GSoC and others decide it for us, but for NumFOCUS grants, grants directly from pandas funds, or corporate partnerships, would be useful to know.

File: SURVEY.md [hunk 3]
```diff
@@ -4,121 +4,53 @@
...
-- 2–10
-- 11–50
-- 51–500
-- 501–1,000
-- 1,001–5,000
-- More than 5,000
-- Not sure
-
-### Experience in IT
+### 2.3. For how long have you kept this job role?
```
Comment (datapythonista) [inline SURVEY.md:51]: For my interests, I find more relevant the previous question or total years of experience (with pandas, python or in IT in general). Happy to also have this, but I'd like to have the others.

And I'm not sure if it's useful to have bins, all the questions with numerical values (age, years of experience...) can probably ask for the exact value.

**Review by datapythonista — APPROVED** [review]
Looks great, couple of comments, but the changes here are great, so happy to get this merged and continue with other changes in a follow up.

File: SURVEY.md (hunk 2 above)
Comment (martinagvilas) [reply SURVEY.md:175]: great to know! I can add them to this PR

File: SURVEY.md (hunk 3 above)
Comment (martinagvilas) [reply SURVEY.md:51]: Ok, makes sense! I will re-add the deleted questions. I also agree with the age values, I can change the output of these questions as well.

**Comment by martinagvilas** [comment]
I added the suggestions, let me know if this looks better now.

**Review by datapythonista — APPROVED** [review]
Looks great. I guess people may still want contribute things to the survey, but surely better in follow ups that in comments here.

File: SURVEY.md [hunk 4]
```diff
@@ -147,12 +83,105 @@ Add all the questions that pops in your mind we will rank them later on.**
...
+- yes
+- no
 
+### 5.2. If you answered yes in question (5.1.), please give reasons for why you should receive a scholarship.
+- (open question)
 
+### 5.3. How many hours per week could you spend working in this internship?
+- (open question)
 
+### 5.4. What amount of money do you expect to receive from this scholarship?
```
Comment (sara-02) [inline SURVEY.md]: `...amount(USD $)...`

File: SURVEY.md [hunk 5]
```diff
@@ -147,12 +83,105 @@ Add all the questions that pops in your mind we will rank them later on.**
...
+
+### 4.1. On average how often do you do the following? (5 = More than once a day; 4 = Once a day; 3 = Every other day; 2 = Once a week; 1 = Once a month)
+- Read new issues and PRs in the project's repository
+- Comment other people's issues and PRs in the project's repository
+- Submit new PR to the project's repository
+
+### 4.2. Why did you choose to get involved in the pandas-mentoring program?
+- (open question)
 
+### 4.3. Is this the first open-source project you contribute to?
```
Comment (sara-02) [inline SURVEY.md]: `...you are contributing...`

File: SURVEY.md [hunk 6]
```diff
@@ -147,12 +83,105 @@ Add all the questions that pops in your mind we will rank them later on.**
...
+- Network programming
+- Software prototyping
+- DevOps / System administration / Writing automation scripts
+
+### 3.4. For how long have you been using pandas?
+- (open question)
+
+### 3.5. For how long have you been using python?
+
+### 3.6. Which of pandas capabilities are most important to you?
```
Comment (sara-02) [inline SURVEY.md]: `...of the following...`

**Review by sara-02 — CHANGES_REQUESTED** [review]
Minor improvements/suggestion in language.

**Comment by sara-02** [comment]
@martinagvilas Minor nitpicking, just to improve the language.

**Comment by martinagvilas** [comment]
@sara-02 great! I made the changes you suggested, let me know if it looks good now.

**Review by sara-02 — APPROVED** [review]
LGTM

**Comment by sara-02** [comment]
@martinagvilas We are good.

**Pull request merged**